sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/Models/AzureDataExplorerLinkedService.cs

```diff
@@ -41,24 +41,24 @@ public AzureDataExplorerLinkedService()
         /// https://&lt;clusterName&gt;.&lt;regionName&gt;.kusto.windows.net.
         /// Type: string (or Expression with resultType string)</param>
-        /// <param name="servicePrincipalId">The ID of the service principal
-        /// used to authenticate against Azure Data Explorer. Type: string (or
-        /// Expression with resultType string).</param>
-        /// <param name="servicePrincipalKey">The key of the service principal
-        /// used to authenticate against Kusto.</param>
         /// <param name="database">Database name for connection. Type: string
         /// (or Expression with resultType string).</param>
-        /// <param name="tenant">The name or ID of the tenant to which the
-        /// service principal belongs. Type: string (or Expression with
-        /// resultType string).</param>
         /// <param name="additionalProperties">Unmatched properties from the
         /// message are deserialized this collection</param>
         /// <param name="connectVia">The integration runtime reference.</param>
         /// <param name="description">Linked service description.</param>
         /// <param name="parameters">Parameters for linked service.</param>
         /// <param name="annotations">List of tags that can be used for
         /// describing the linked service.</param>
-        public AzureDataExplorerLinkedService(object endpoint, object servicePrincipalId, SecretBase servicePrincipalKey, object database, object tenant, IDictionary<string, object> additionalProperties = default(IDictionary<string, object>), IntegrationRuntimeReference connectVia = default(IntegrationRuntimeReference), string description = default(string), IDictionary<string, ParameterSpecification> parameters = default(IDictionary<string, ParameterSpecification>), IList<object> annotations = default(IList<object>))
+        /// <param name="servicePrincipalId">The ID of the service principal
+        /// used to authenticate against Azure Data Explorer. Type: string (or
+        /// Expression with resultType string).</param>
+        /// <param name="servicePrincipalKey">The key of the service principal
+        /// used to authenticate against Kusto.</param>
+        /// <param name="tenant">The name or ID of the tenant to which the
+        /// service principal belongs. Type: string (or Expression with
+        /// resultType string).</param>
+        public AzureDataExplorerLinkedService(object endpoint, object database, IDictionary<string, object> additionalProperties = default(IDictionary<string, object>), IntegrationRuntimeReference connectVia = default(IntegrationRuntimeReference), string description = default(string), IDictionary<string, ParameterSpecification> parameters = default(IDictionary<string, ParameterSpecification>), IList<object> annotations = default(IList<object>), object servicePrincipalId = default(object), SecretBase servicePrincipalKey = default(SecretBase), object tenant = default(object))
```

Comment:
this will result in a breaking change, please have the breaking change approval board add the "Approved-BreakingChange" label to https://github.com/Azure/azure-rest-api-specs/pull/12755

Reply:
@m-nash This change has been reverted in this PR https://github.com/Azure/azure-rest-api-specs/pull/12769, Could you review this again?